test-case-reporting/app.ts

```diff
@@ -68,8 +69,20 @@ function extractPageInfo(req: express.Request): PageInfo {
   };
 }
 
-app.get('/', (req, res) => {
-  res.send('Hello, world, 2.0!');
+app.get(['/', '/builds'], async (req, res) => {
+  const {json} = req.query;
+
+  try {
+    const builds = await record.getRecentBuilds(extractPageInfo(req));
+
+    if (json) {
+      res.json({builds});
+    } else {
+      res.send(render('build-list', {title: 'Latest Builds', builds}));
```

Comment:
This title field is a good pattern to use for the test run list pages as well, so you could set the title as "Test History - 'the thing...'" or "Test Runs - Build #12345" etc.

Reply:
(feel free to punt this to a later PR)